tests/gtest/avifcllitest.cc

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Google LLC
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "avif/avif.h"
+#include "aviftest_helpers.h"
+#include "gtest/gtest.h"
+
+namespace libavif {
+namespace {
+
+TEST(ClliTest, Simple) {
+  testutil::AvifImagePtr image =
+      testutil::CreateImage(/*width=*/8, /*height=*/8, /*depth=*/8,
+                            AVIF_PIXEL_FORMAT_YUV444, AVIF_PLANES_YUV);
+  ASSERT_NE(image, nullptr);
+  testutil::FillImageGradient(image.get());  // The pixels do not matter.
+
+  for (int max_content_light_level : {0, 1, 65535}) {
+    for (int max_pic_average_light_level : {0, 1, 65535}) {
```

Comment:
Should we declare the loop index variables as `uint16_t` to avoid the casts in lines 20-21?

Reply:
I found the answer to my question: the initializer list `{0, 1, 65535}` has the element type `int`, and when we assign an `int` value to a `uint16_t` variable (either the loop index variable or the clli struct members) we need an explicit cast to avoid the C4244 "possible loss of data" warning under MSVC.